main/zoe/api/zoe.md

```diff
@@ -3,84 +3,299 @@
+- Returns: `{Promise<StartInstanceResult>}`
+
+Create an `instance` of the installed smart contract (specified by 
+the `installation` argument). You must also specify the 
+instance's `issuerKeywordRecord` and `terms` for the contract 
+(as key-value pairs). 
+
+The `issuerKeywordRecord` is a record mapping string names (keywords) 
+to `issuers`, such as `{ Asset: quatlooIssuer}`. Keywords must begin 
+with a capital letter and must be ASCII. Parties to the contract will 
+use the keywords to index their proposal and their payments. 
+
+`terms` are values used by this contract instance, such as the 
+number of bids an auction will wait for before closing. These values may
+be different for different instances of the same contract, but the contract
+defines what variables need their values passed in as `terms`. 
+
+It returns a `promise` for a `StartInstanceResult` object. The object consists of:
+- `creatorFacet` `{any}`
+- `publicFacet` `{any}`
+- `instance` `{Instance}`
+- `creatorInvitation `{Payment | undefined}`
+
+**tyg todo: How is the creatorInvitation used?**
 ```js
 const issuerKeywordRecord = { 
   'Asset' : moolaIssuer, 
-  'Price' : simoleanIssuer 
+  'Price' : quatlooIssuer 
 };
 const terms = { numBids: 3 };
-const { invite, instanceRecord } = await E(zoe).makeInstance(
-  secondPriceAuctionInstallationHandle, 
+const { invite, instanceRecord } = await E(zoe).startInstance(
+  auctionInstallationHandle, 
   issuerKeywordRecord, 
   terms
 );
 ```
+## E(Zoe).offer(invitation, proposal, paymentKeywordRecord)
+- `invitation` `{Invitation|Promise<Invitation>}`
+- `proposal` `{Proposal}`
+- `paymentKeywordRecord` `{PaymentKeywordRecord}`
+- Returns: `{Promise<UserSeat>}`
+
+Used to exercise the `invitation` provided as the first argument.
+
+To redeem (sometimes called "exercise") an `invitation`, a user normally provides a `proposal` (their
+rules for the offer) as well as `payments` to be escrowed by Zoe.  If
+either the `proposal `or `payments` are empty, indicate this by
+omitting that argument or passing `undefined`, instead of passing an
+empty record.
+
+The `proposal` has three parts: 
+- `want`: An object with keywords as keys and amounts as values.
+- `give`: An object with keywords as keys and amounts as values.
+- `exit`: Specifies the payout-liveness policy Zoe can guarantee for the offer.
+
+`paymentKeywordRecord` is a record with keywords as keys, with
+ values of the actual `payments` to be escrowed. A `payment` is
+ expected for every rule under `give`.
+ 
+ `offer()` returns a `promise` for a `userSeat`. See the Objects section for its
+ description.
+ ```js
+ **tyg todo: Need good source code**
+ ```
+
+
+---------------------------
+**tyg todo: As mentioned in the standup, I think we now need to cover
+API objects in the doc. I'm undecided if they should be one page with methods,
+on have their own pages. Thoughts? For the time being, I'm including them on the
+same page with a h1 heading**
+
+# Objects
```

Comment:
I've got two concerns about this:

1) Findability. If objects are described within a method's documentation, the indexing is going to be on the method. If the object doesn't have effectively the same name as the method, it'll be hard to find. If an object is used at all outside of its immediate creation by a method, it should have its own, indexable and findable, entry.

2) Length. Some of these objects, particularly when I add on method documentation for methods on them, are already going to be pretty long in docs. Adding them to method entries will make for several very long entries. On the other hand, people are mostly likely to use the reference one (method) entry at a time, so this may not be a significant issue. 

If we're going to intermingle method and object writeups, I'd prefer it be of the form:

Method Entry with ## title to index from   

Related to above method Object Entry with ## to index from
Additional related to above method Object Entry with ## to index from

Next Method ## Entry.

Reply:
As discussed elsewhere, we are not doing an object section, and are instead moving the objects into the various sections in which they are introduced/used, but are collapsing the object part such that it doesn't take up too much room. Additionally, h1, h2, and h3, all show up in the sidebar, so we can use `###` headers for the objects and they should still show up in the sidebar.